docs/transactions/iti57.md

```diff
@@ -1,11 +1,30 @@
 # ITI-57
 
-ITI-57: Update Document Set transaction is described in *IHE IT Infrastructure
-Technical Framework Supplement XDS Metadata Update*.
+The [ITI-57 (Update Document Set)](https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_XDS_Metadata_Update.pdf) transaction allows a _Document Administrator_ (see referenced documentation, appendix A) to update existing documents. 
```

Comment:
Even though the IHE documentation refers to the _document administrator_ actor, I think in this context this would be confusing, since this transaction is not restricted to PMP document administrators.